java/src/org/openqa/selenium/json/JsonOutput.java

```diff
@@ -110,159 +112,169 @@ public class JsonOutput implements Closeable {
   private String indentBy = "  ";
   private boolean writeClassName = true;
 
+  /**
```

Comment:
Unless it's really helpful to people modifying the code, we only need to document `public` and `protected` methods.

Reply:
I've removed the JavaDoc headers from all private and package-private methods that don't seem to need them.